### PR TITLE
Unlocked blackmarket loot, Blackjack specialist, mega gobblegums, extra CaC slots, CWL camo's and paid specialist outfits

### PR DESCRIPTION
**Pull request by JariKCoding**

### CODE REVIEW

File: src/client/component/loot.cpp [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#include <std_include.hpp>
+#include "loader/component_loader.hpp"
+
+#include <utils/hook.hpp>
+#include <utils/thread.hpp>
+
+#include <component/scheduler.hpp>
```
Comment (diamante0018) [inline src/client/component/loot.cpp:7]: Only `schduler.hpp` will work

File: src/client/component/loot.cpp [hunk 2]
```diff
@@ -0,0 +1,62 @@
+#include <std_include.hpp>
+#include "loader/component_loader.hpp"
+
+#include <utils/hook.hpp>
+#include <utils/thread.hpp>
+
+#include <component/scheduler.hpp>
+#include "game/game.hpp"
```
Comment (diamante0018) [inline src/client/component/loot.cpp:8]: Usually, in other momo's project `game` header is put at the top not bottom (after `#include "loader/component_loader.hpp"`)
The reason is that when you include other components that require game definition you will get compilation error if game.hpp is not included at the top.

**Review by diamante0018 — COMMENTED** [review]
Very interesting pr 😀

File: src/client/component/loot.cpp (hunk 2 above)
Comment (momo5502) [reply src/client/component/loot.cpp:8]: While this is true, the reason is not entirely correct. The reason this is the case is that I don't really have a header preference. Ideally, each header should compile on its own, meaning it whould be irrelevant if you include game before or after other headers.

**Review by momo5502 — APPROVED** [review]

File: src/client/component/loot.cpp (hunk 2 above)
Comment (JariKCoding) [reply src/client/component/loot.cpp:8]: Thanks for the feedback! I'll keep that in mind in the future.

**Pull request merged**